dgraph/cmd/bulk/reduce.go

```diff
@@ -130,7 +130,7 @@ func (r *reducer) createBadgerInternal(dir string, compression bool) *badger.DB
 
 	opt := badger.DefaultOptions(dir).WithSyncWrites(false).
 		WithTableLoadingMode(bo.MemoryMap).WithValueThreshold(1 << 10 /* 1 KB */).
-		WithLogger(nil).WithMaxCacheSize(1 << 20).
+		WithLogger(nil).WithBlockCacheSize(1 << 20).
```

Comment:
shouldn't we use the flag configured instead of 1 <<  20 ?

Reply:
I think we should. That would require adding a flag in `dgraph bulk`. @jarifibrahim  What do you say?

Reply:
Bulk loader shouldn't even have a cache. I'll create a ticket for this. This Pr doesn't need to block on this.

Reply:
I see that those tmpDBs are being read. We need a cache here but it has to be configurable. I have created DGRAPH-2352